_ingest-pipelines/processors/set.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Set
+parent: Ingest processors
+nav_order: 240
+---
+
+# Set processor
+
+The `set` processor is used to <explain what is used to do>.
+
+The following is the syntax for the `set` processor:
+
+```json
+<insert syntax example>
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `set` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
```

Comment:
@gaobinlong Please confirm all parameters are correct and listed for this processor.